scripts/rspec/rspec.ps1

```diff
@@ -267,20 +267,31 @@ function AppendVbTestCase($ruleTestsFolder) {
     $existingClassText = Get-Content -Path "${ruleTestsFolder}\\${csClassName}Test.cs" -Raw
     $snippetText = Get-Content -Path "${RuleTemplateFolder}\\VbNetTestSnippet.cs" -Raw
 
-    $namespaceToken = "using CS = SonarAnalyzer.Rules.CSharp;"
+    $usingToken = "using SonarAnalyzer.Rules.CSharp;"
+    $csUsingToken = "using CS = SonarAnalyzer.Rules.CSharp;"
```

Comment:
You know the value of `${csClassName}`. So you can try to replace `new ${csClassName}` with `new CS.${csClassName}` to fix the usages.